docs/tutorials/odc-stac.rst

```diff
@@ -0,0 +1,290 @@
+
+| Once launched, you should see a Jupyter notebook environment with the tutorial notebook open. Open the file browser by clicking the folder icon on the left-hand menu bar (circled in red in the image below).
+
+.. image:: ../_static/tutorial-images/odc-stac/binder_view.png
+ :width: 600
+ :alt: A view of a Jupyter notebook environment showing that the tutorial.ipynb file is loaded. The file is open with the header "odc-stac tutorial".
+ :align: center
+
+| The tutorial notebook has headers that match up with the tutorial instructions below.
+
+.. note::
+   For this tutorial, we believe you will learn more if you type the code yourself, rather than using copy-paste.
+   Typing encourages you to slow down and think about what you're doing, which will help you gain understanding!
+
+   If you are stuck at any point, open the :code:`tutorial_solution.ipynb` notebook that is available in the file browser.
+
+Tutorial
+========
+
+Python imports
+--------------
+
+The notebook requires five libraries to run:
+
+* :code:`geopandas` for loading an area of interest from a 
+* :code:`odc.geo` for exporting loaded data
+* :code:`odc.stac` for loading data
+* :code:`planetary_computer` to provide authentication when accessing data
+* :code:`pystac-client` for querying catalogs of data
+
+We will import either the library, or specific functions and classes from the library.
+Type the following into the empty cell below the **Python imports** heading:
+
+.. code-block:: python
+
+   import geopandas as gpd
+   from odc.geo.xr import write_cog
+   from odc.stac import load
+   import planetary_computer
+   from pystac_client import Client
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+   
+Set up query parameters
+-----------------------
+
+In this section of the tutorial, you will specify:
+
+* The area you want to load data for
+* The date range you want to load data for
+* The data source you want to load from
+
+Area of interest
+^^^^^^^^^^^^^^^^
+
+We specify the area of interest using the :code:`aoi.geojson` file, which can be loaded with :code:`geopandas`.
+
+The area of interest is the southern part of Isla Isabela, one of the islands in the Galapagos.
+
+.. image:: ../_static/tutorial-images/odc-stac/aoi.png
+ :width: 600
+ :alt: A satellite image of Isla Isabela, with the area of interest shown as a yellow bounding box.
+ :align: center
+
+| Type the following into the empty cell below the **Area of interest** heading:
+
+.. code-block:: python
+
+   aoi = gpd.read_file("aoi.geojson")
+   aoi_geometry = aoi.iloc[0].geometry
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+
+Date range
+^^^^^^^^^^
+
+We must specify a start and end date for our query.
+Type the following into the empty cell below the **Date range** heading:
+
+.. code-block:: python
+
+   start_date = "2017-01-01"
+   end_date = "2023-01-01"
+   date_query = (start_date, end_date)
```

Comment:
The heading is "Date range" and the way I read the text implies a date range, so those two seem aligned. Continuing from that buildup into the code and then not naming the variable `date_range` is a bit surprising to me.

Reply:
Have changed this to `date_range` now